### PR TITLE
Adds admin message when canister with BZ is open

### LINKED ISSUE #18605: BZ cans don't make an admin message when opened
Unlike N2O or Plasma.

### PR DESCRIPTION
**Pull request by lzimann**
Fixes #18605

### CODE REVIEW

File: code/modules/atmospherics/machinery/portable/canister.dm [hunk 1]
```diff
@@ -307,9 +307,10 @@
 				if(!holding)
 					var/plasma = air_contents.gases["plasma"]
 					var/n2o = air_contents.gases["n2o"]
-					if(n2o || plasma)
-						message_admins("[key_name_admin(usr)] (<A HREF='?_src_=holder;adminmoreinfo=\ref[usr]'>?</A>) (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[usr]'>FLW</A>) opened a canister that contains [n2o ? "N2O" : ""][n2o && plasma ? " & " : ""][plasma ? "Plasma" : ""]! (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)")
-						log_admin("[key_name(usr)] opened a canister that contains [n2o ? "N2O" : ""][n2o && plasma ? " & " : ""][plasma ? "Plasma" : ""] at [x], [y], [z]")
+					var/bz = air_contents.gases["bz"]
+					if(n2o || plasma || bz)
+						message_admins("[key_name_admin(usr)] (<A HREF='?_src_=holder;adminmoreinfo=\ref[usr]'>?</A>) (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[usr]'>FLW</A>) opened a canister that contains [n2o ? "N2O" : ""][n2o && plasma && bz ? " & " : ""][plasma ? "Plasma" : ""][bz ? "BZ" : ""]! (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)")
```
Comment (Supermichael777) [inline code/modules/atmospherics/machinery/portable/canister.dm]: uhh this will now error with some combos

Comment (lzimann) [inline code/modules/atmospherics/machinery/portable/canister.dm]: welp, gonna change it later then

**Comment by optimumtact** [comment]
@lzimann

**Comment by lzimann** [comment]
should be good now.

**Pull request merged**